module/src/components/codeInput/codeInput.component.tsx

```diff
@@ -313,22 +320,42 @@ export const CodeInput = React.forwardRef(
             pending={pending}
             validationMode={bindConfig.validationMode}
           >
-            <IconWrapper leftIcon={leftIcon} rightIcon={rightIcon}>
+            <>
+              {leftIcon && (
+                <>
+                  {IconUtils.isIconDefinition(leftIcon) ? (
+                    <Icon {...leftIcon} />
```

Comment:
This needs the title `[iconname] icon on left` and the equivalent for the right icon, and everywhere these are used

Reply:
Added everywhere where its used